src/index.html

```diff
@@ -5,8 +5,273 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img 
+        src="images/logo.png" 
+        alt="moyo_logo" 
+        class="logo"
+      >
+      </img>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="https://apple.com" 
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://samsung.com" 
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://smartphones.com" 
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://laptops.com" 
+              class="nav__link" 
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://gadgets.com" 
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://tablets.com" 
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://photo.com" 
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="https://video.com" 
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <article class="card catalog__card" data-qa="card">
```

Comment:
data-qa should be set only in the firs card
![image](https://user-images.githubusercontent.com/86923089/207693981-8cbafb41-2f00-4d19-bcd8-e7e32e3b2fb3.png)